### PR TITLE
DROOLS-2076: [DataModelOracle] DSL Sentences are not loaded

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/DROOLS-2076

### CODE REVIEW

**Comment by manstis** [comment]
Part of an ensemble:

- https://github.com/kiegroup/kie-soup/pull/10
- https://github.com/kiegroup/drools-wb/pull/679
- https://github.com/kiegroup/kie-wb-distributions/pull/647

**Comment by manstis** [comment]
Depends on https://github.com/kiegroup/drools/pull/1557 to test.

File: kie-soup-project-datamodel/kie-soup-project-datamodel-commons/src/main/java/org/kie/soup/project/datamodel/commons/oracle/PackageDataModelOracleImpl.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ public void addExtensions(Map<ExtensionKind<?>, List<?>> packageElements) {
         packageElements
                 .forEach((k, v) -> {
                     @SuppressWarnings("rawtypes")
-                    List list = packageElements.computeIfAbsent(k, k1 -> new ArrayList<>());
+                    List list = this.packageElements.computeIfAbsent(k, k1 -> new ArrayList<>());
```
Comment (jomarko) [inline kie-soup-project-datamodel/kie-soup-project-datamodel-commons/src/main/java/org/kie/soup/project/datamodel/commons/oracle/PackageDataModelOracleImpl.java]: I think worth to cover with simple unit test. Let me know what do you think @manstis.

Comment (manstis) [reply kie-soup-project-datamodel/kie-soup-project-datamodel-commons/src/main/java/org/kie/soup/project/datamodel/commons/oracle/PackageDataModelOracleImpl.java]: @jomarko Unit Test added in new commit.

Comment (jomarko) [reply kie-soup-project-datamodel/kie-soup-project-datamodel-commons/src/main/java/org/kie/soup/project/datamodel/commons/oracle/PackageDataModelOracleImpl.java]: Thanks, looks good. Will approve the ensemble when tried manually.

**Comment by mbarkley** [comment]
Thank you for fixing this @manstis.

**Pull request merged**